integration/actions_test.go

```diff
@@ -70,6 +70,33 @@ func chain(actions ...action) action {
 	}
 }
 
+// chdir changes the test working directory to the subdirectory for the duration of the action.
+func chdir(dir string, a action) action {
+	return func(t testing.TB, ic testContext) error {
+		dir := filepath.Join(ic.workDir, dir)
+		infof("Changing directory to %s", dir)
+		cwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		ic.workDir = dir
+		err = os.Chdir(dir)
+		if err != nil {
+			return err
+		}
+		defer os.Chdir(cwd)
+		return a(t, ic)
+	}
+}
+
+// debugShell opens a new Terminal window in the test working directory.
+func debugShell() action {
```

Comment:
Handy for debugging why the hell a test action is failing, as the test working directory gets wiped after the test.